tests/contrib/uwsgi/__init__.py

```diff
@@ -0,0 +1,20 @@
+import os
+import subprocess
+import sys
+
+
+def run_uwsgi(cmd):
+    def _run(*args):
+        env = os.environ.copy()
+        if sys.version_info[0] == 2:
+            # On Python2, it's impossible to import uwsgidecorators without this hack
+            env["PYTHONPATH"] = os.path.join(
```

Comment:
instead of overwrite, should we append to any existing `PYTHONPATH`?

e.g. `PYTHONPATH="${PYTHONPATH}:venv/lib/python2.7/site-packages/` ?